src/main/java/Hw_22/InMemoryImpl.java

```diff
@@ -0,0 +1,20 @@
+package Hw_22;
+
+import java.util.*;
+
+public class InMemoryImpl implements FitnessTracker {
+    private  ArrayList<Object> stringArrayList = new ArrayList<>();
+
+    @Override
+    public int saveData(String fileName, Object... data) {
```

Comment:
What for fileName parameter in InMemotyImp? I suggest to remove it from parent interface